src/feature/header/burger-menu.tsx

```diff
@@ -63,9 +67,11 @@ export default function BurgerMenu() {
           </SheetDescription>
           <div className="bg-background flex items-center justify-between gap-10 p-4">
             <div className="flex w-24 cursor-pointer items-center justify-center">
-              <NavLink to="/">
-                <img src="logo.svg" alt="logo" className="rounded-2xl bg-white" />
-              </NavLink>
+              <SheetClose asChild>
+                <NavLink to="/">
+                  <img src="logo.svg" alt="logo" className="rounded-2xl bg-white" />
```

Comment:
```suggestion
                  <img src="/logo.svg" alt="logo" className="rounded-2xl bg-white" />
```
Надо добавлять `/` перед именем, чтоб находясь на адресе `poster-store.com/**/***` картинка не терялась


![image](https://github.com/user-attachments/assets/d49f9bbe-9744-4b9f-ab49-09ee2c946af5)

Reply:
спасибо за комменты :four_leaf_clover: